scripts/SANS/sans/gui_logic/models/sum_runs_model.py

```diff
@@ -5,30 +5,24 @@
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 import SANSadd2
-from ui.sans_isis.work_handler import WorkHandler
+from mantidqt.utils.async_qt_adaptor import qt_async_task, IQtAsync
 
 
-class SumRunsModel(object):
-    def __init__(self, work_handler, view=None):
-        self._work_handler = work_handler
+class SumRunsModel(IQtAsync):
+    def __init__(self, view=None):
+        super().__init__()
+        # TODO this should be in presenter, not the model
```

Comment:
It depends:
- If your happy for a much larger diff I can move the tests and slots to the presenter 
- Equally I won't create an issue if we want to leave this, because I'll pick it up next time I touch the file (continuous refactoring rather then dedicated)

I'm happy to do either?

Reply:
I'm going to leave this for the future when I next touch anything related to Sum Runs